lib/cloudex.ex

```diff
@@ -54,12 +54,12 @@ defmodule Cloudex do
   @spec upload_list_with_options(list(map), map) :: upload_result
   def upload_list_with_options(list, default_options \\ %{}) do
     list
-    |> Enum.map(fn %{url: url} = item ->
-      item_specific_options = Map.drop(item, :url)
+    |> Enum.map(fn %{image_resource: image_resource} = item ->
```

Comment:
@chrisplim I prefer to keep url, it is more precise wording

Reply:
it's more accurate to have image_resource, since this is not limited to urls. it can be files, uris or urls (subset of uri)

Reply:
but then what if it is a video?

Reply:
true, i'll just label it resource